GooglePlayInstant/LoadingScreen/Scripts/LoadingScreen.cs

```diff
@@ -78,10 +78,7 @@ private IEnumerator AttemptAssetBundleDownloadsCo(int numberOfAttempts)
                 _assetBundleRetrievalAttemptCount++;
                 Debug.LogFormat("Attempt #{0} at downloading AssetBundle...", _assetBundleRetrievalAttemptCount);
 
-                while (_bundle == null)
-                {
-                    yield return GetAssetBundle(AssetBundleUrl);
-                }
+                 yield return GetAssetBundle(AssetBundleUrl);
```

Comment:
Nit: indentation is slightly off.

By the way, I can't comment on line 102, but does that final "_downloading = false;" do anything?

Reply:
Most of the time no, but only because LoadSceneAsync destroys LoadingScene when its AsyncOperation finishes. I feel like it's good form to keep it though, because the _downloading flag is meant to be true for the lifetime of that coroutine. There is a theoretical case a developer marks the LoadingScene as DontDestroyOnLoad, in which case the LoadingScene would be put in an invalid state. Thats very unlikely though, so I'm open to deleting it if you think its unnecessary.

Reply:
OK as is. Thanks for the explanation.